docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,184 @@
+  rules without a license, the CLI groups all errors and reports a
+  single warning.
+* The project name for repositories scanned locally is `local_scan/<repo_name>`
+  instead of `<repo_name>`.
+* The **View Results** URL displayed for findings now includes the repository
+  and branch names.
+
+### Fixed
+
+* Fixed an issue with incorrect autofix application where multiple fixes were
+  applied to the same line.
+* Fixed issue where tokens for type parameter brackets weren't stored correctly.
+  They're now stored in the generic AST, allowing Semgrep to autofix
+  these constructs correctly.
+* Fixed an issue where Semgrep doesn't support multiple labels for taint
+  traces. Now, Semgrep looks at the `requires` of the sink, and if it has the
+  shape `A and ...`, it picks `A` as the preferred label and reports the
+  trace.
+* Fixed issue where taint signatures don't capture changes to parameter fields.
+
+## ⛓️ Supply Chain
+
+### Added
+
+* Added support for parsing Swift Package Manager manifest files and lock files.
+* Added the ability to filter for dependencies that Semgrep has commented on.
+  <!-- https://github.com/semgrep/semgrep-app/pull/12898 -->
+* Added manual review advice to GitHub PR comments. Certain Semgrep Supply Chain (SSC) findings require **manual review** to verify if the finding is reachable or not. GitHub PR comments now include this advice to help you ascertain if the finding is reachable or not. <!-- 12907 -->
+* Separated `reachable_if` and `upgrade_only` exposures. Users can better determine a finding's criticality through this distinction. <!-- 12657 -->
+
+### Fixed
+
+* Fixed issues with trailing newline parsing in `pyproject.toml` and
+  `poetry.lock` files.
+
+## 🤖 Assistant (beta)
+
+### Added
+
+- Added weekly emails summarizing Semgrep tasks, highlighting top priority items. <!-- Private beta - not sure if we should include this --> 
+
+## 🔐 Secrets (beta)
+
+### Added
+
+- Added the following new rules:
+  - Detection rules for Azure and AWS
+  - Semantic secrets rules for Python, JavaScript, and TypeScript
+  - Semantic rules for hard-coded credentials in bash for `curl` commands
+- Added non-validator regex detection for databases, including MongoDB,
+  Microsoft SQL Server, MySQL, Postgres, and Redis
+- Added secrets rule management, which is accessible in Semgrep Cloud Platform
+  by going to **Rules** > **Policies** > **Secrets**. This allows you to:
+  - See all available rules
+  - Set valid finding modes for the rules
+  - Set invalid and error validation state modes across multiple rules
+
+### Fixed
+
+- Fixed an issue where the **Analysis method** filter in Semgrep Cloud Platform
+  wasn't filtering correctly.
+
+## 📝 Documentation and knowledge base
+
+### Added
+
+- The Semgrep docs sidebar has been reorganized to help users browse through the docs.
+- A [series of guides](/deployment/core-deployment/) to setting up Semgrep as part of a security program for your organization.
```

Comment:
Maybe add "Added..." to the beginning of each entries? I think this verb could be omitted if we didn't have the first sentence mentioning the sidebar reorg, though

Reply:
I will add "added"!